server/views/referrals/courseParticipations/details.njk

```diff
@@ -0,0 +1,172 @@
+  id: "custodyLocation",
+  name: "setting[location]",
+  classes: "govuk-!-width-one-third",
+  label: {
+    text: "Enter the prison (if known)"
+  }
+}) }}
+{% endset %}
+
+{% set completeOutcomeHtml %}
+{{ govukInput({
+  id: "yearCompleted",
+  name: "outcome[yearCompleted]",
+  inputmode: "numeric",
+  spellcheck: false,
+  classes: "govuk-!-width-one-third",
+  label: {
+    text: "Enter the year completed (if known)"
+  }
+}) }}
+{% endset %}
+
+{% set incompleteOutcomeHtml %}
+{{ govukInput({
+  id: "yearStarted",
+  name: "outcome[yearStarted]",
+  inputmode: "numeric",
+  spellcheck: false,
+  classes: "govuk-!-width-one-third",
+  label: {
+    text: "Enter the year started (if known)"
+  }
+}) }}
+{% endset %}
+
+{% block content %}
+  <h1 class="govuk-heading-l">{{ pageHeading }}</h1>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <form action="#" method="post">
+        <input type="hidden" name="_csrf" value="{{ csrfToken }}"/>
+
+        {{ govukRadios({
+          name: "setting[type]",
+          fieldset: {
+            legend: {
+              text: "Select the setting"
+            }
+          },
+          items: [
+            {
+              value: "community",
+              text: "Community",
+              conditional: {
+                html: communitySettingHtml
+              },
+              attributes: {
+                "data-testid": "community-setting-option"
+              }
+            },
+            {
+              value: "custody",
+              text: "Custody",
+              conditional: {
+                html: custodySettingHtml
+              },
+              attributes: {
+                "data-testid": "custody-setting-option"
+              }
+            }
+          ],
+          attributes: {
+            "data-testid": "setting-options"
+          }
+        }) }}
+
+        {{ govukRadios({
+          name: "outcome[status]",
+          fieldset: {
+            legend: {
+              text: "Select the outcome (if known)"
+            }
+          },
+          items: [
+            {
+              value: "complete",
+              text: "Complete",
+              conditional: {
+                html: completeOutcomeHtml
+              },
+              attributes: {
+                "data-testid": "complete-outcome-option"
+              }
+            },
+            {
+              value: "incomplete",
+              text: "Incomplete",
+              conditional: {
+                html: incompleteOutcomeHtml
+              },
+              attributes: {
+                "data-testid": "incomplete-outcome-option"
+              }
+            }
+          ],
+          attributes: {
+            "data-testid": "outcome-options"
+          }
+        }) }}
+
+        {{ govukTextarea({
+          name: "outcome[detail]",
+          id: "outcomeDetail",
+          label: {
+            text: "Can you provide additional detail? (if known)"
```

Comment:
Looks like in the V7 designs this should be "Provide additional detail (if known)"